pkg/image/airgap.go

```diff
@@ -162,7 +174,20 @@ func TagAndPushImagesFromBundle(airgapBundle string, options imagetypes.PushImag
 		if err := tarGz.Unarchive(airgapBundle, extractedBundle); err != nil {
 			return errors.Wrap(err, "falied to unarchive airgap bundle")
 		}
-		return PushImagesFromTempRegistry(extractedBundle, airgap.Spec.SavedImages, options)
+		if err := PushImagesFromTempRegistry(extractedBundle, airgap.Spec.SavedImages, options); err != nil {
+			return errors.Wrap(err, "failed to push images from temp registry")
+		}
+
+		pushEmbeddedArtifactsOpts := imagetypes.PushEmbeddedArtifactsOptions{
+			Registry:   options.Registry,
+			Tag:        imageutil.SanitizeTag(fmt.Sprintf("%s-%s-%s", airgap.Spec.ChannelID, airgap.Spec.UpdateCursor, airgap.Spec.VersionLabel)),
+			HTTPClient: orasretry.DefaultClient,
+		}
+		if err := PushEmbeddedClusterArtifacts(extractedBundle, pushEmbeddedArtifactsOpts); err != nil {
```

Comment:
airgap bundle format for images doesn't actually effect how to process embedded cluster artifacts as they're outside the airgap bundle registry. how about moving this outside the `switch` statement?

Reply:
this code path is the only one where the bundle is extracted, so it seemed to fit in there best. it looks like it would require additional refactoring to move it outside of the `switch` statement

Reply:
the suggested change is to have `PushEmbeddedClusterArtifacts` accept a compressed bundle archive and do it's own extraction, as necessary